documentation/source/simulator_support.rst

```diff
@@ -102,7 +102,7 @@ To enable :term:`HDL` code coverage, add Verilator's coverage option(s) to the :
 
     EXTRA_ARGS += --coverage
 
-This will result in coverage data being written to ``coverage.dat``.
+This will result in coverage data being written to :file:`coverage.dat`.
```

Comment:
I think this will try to link against the non-existent file coverage.dat. Best to leave it where it is.

Reply:
To me, the support for curly braces to support variable parts (https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-file) suggests that this role is meant for semantic markup, not necessarily to reference existing files; at least that seemed to be the initial idea.

Reply:
I see. I am seeing warnings related to the referencing of options that weren't declared and assumed this would have a similar issue.